content/blog/newsletter/2025-07-05-newsletter.de.md

```diff
@@ -0,0 +1,198 @@
+- [Introducción a XMPP - Capítulo 2 - OMEMO](https://jdrm.info/introduccion-a-xmpp-capitulo-2-omemo.html) von [Jacobo Da Riva Muñoz](https://jdrm.info/pages/about.html). [ES]
+- [Introducción a XMPP - Capítulo 3 - Grupos](https://jdrm.info/introduccion-a-xmpp-capitulo-3-grupos.html) von [Jacobo Da Riva Muñoz](https://jdrm.info/pages/about.html). [ES]
+- [Mensajería segura y abierta mediante CONVERSATIONS Xmpp](https://alt43.es/mensajeria-segura-y-abierta-mediante-conversations-xmpp/) auf [alt43.es](https://alt43.es/). [ES]
+- [Recuperando a soberania e a privacidade da sua comunicação pessoal e corporativa](https://dev.to/isafashiondev/recuperando-a-soberania-e-a-privacidade-da-sua-comunicacao-pessoal-e-corporativa-id3) von [Isadora Ribeiro](https://dev.to/isafashiondev). [PT_BR]
+- [Mitigating MITMs in XMPP](https://blog.jmp.chat/b/mitigating-mitms-in-xmpp) von [Amolith](xmpp:amolith@secluded.site) für den [JMP Blog](https://blog.jmp.chat/).
+- [Setting up Slidge Gateway with Openfire for use with Whatsapp, Matrix and Telegram](https://discourse.igniterealtime.org/t/setting-up-slidge-gateway-with-openfire-for-use-with-whatsapp-matrix-telegram/95656/1) von surfbum für die [Ignite Realtime Community](https://www.igniterealtime.org/).
+- [Per-application passwords in Tigase](https://tigase.org/blog/tigase-per-app-credentials/) von [Tigase](https://tigase.org/posts/).
+- [Join the XMPP-IT Community: Your Help Matters!](https://www.xmpp-it.net/2025/07/05/roughnecks/collabora-con-noi/) Die italienische XMPP-Gemeinschaft ruft zur Kollaboration auf – man muss kein:e Expert:in sein, es geht nicht um bezahlte Arbeit, sondern nur um gemeinsame Teilhabe. Von Übersetzungen über das Testen von Software bis hin zu Wiki-Inhalten und Code-Beiträgen, jede:r ist willkommen. [IT]
+
+## Neuigkeiten über XMPP-Software
+
+### XMPP-Clients und -Anwendungen
+
+- [Cheogram](https://cheogram.com/) ist in Version [2.17.10-2](https://git.sr.ht/~singpolyma/cheogram-android/refs/2.17.10-2) für Android erschienen.
+- [Converse.js](https://conversejs.org/) hat Version [11.0.1](https://github.com/conversejs/converse.js/releases/tag/v11.0.1) des quelloffenen, web-basierten XMPP-Chat-Clients veröffentlicht. Die Desktopversion kann [hier](https://github.com/conversejs/converse-desktop/releases) heruntergeladen werden. Hinter dem [Release-Link](https://github.com/conversejs/converse.js/releases) findest Du alle Details.
+- [Gajim](https://gajim.org/) ist in den Versionen [2.3.0](https://gajim.org/posts/2025-06-29-gajim-2.3.0-released/), ([2.3.1 und 2.3.2](https://gajim.org/posts/2025-07-02-gajim-2.3.1-2.3.2-released/)) im neuen [Adwaita](https://gnome.pages.gitlab.gnome.org/libadwaita/)-Look erschienen mit weiteren neuen Feautres, Verbesserungen und ein einigen Änderungen und Fehlerkorrekturen. In ihren [News](https://gajim.org/posts/) gibt es mehr Infos und natürlich im [Changelog](https://dev.gajim.org/gajim/gajim/-/blob/master/ChangeLog)!
```

Comment:
Sorry, we had a commit to the original version. Can you add this? https://github.com/xsf/xmpp.org/pull/1536